tests/compat.sh

```diff
@@ -320,11 +328,13 @@ add_common_ciphersuites()
 
     M_CIPHERS="$M_CIPHERS $CIPHERS"
 
-    G=`python3 scripts/translate_ciphers.py g "$CIPHERS"`
-    G_CIPHERS="$G_CIPHERS $G"
+    T=`python3 scripts/translate_ciphers.py g "$CIPHERS"`
+    check_translation $? $T
```

Comment:
Since `T` is likely to contain whitespace, it should be passed as `"$T"`.

Reply:
ah I see, thanks